examples/kubernetes/dynamic-provisioning/filesystem/storageclass.yaml

```diff
@@ -0,0 +1,28 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: fsx-sc
+provisioner: fsx.openzfs.csi.aws.com
+parameters:
+  volumeType: "filesystem" #REQUIRED
```

Comment:
nit: I feel like it'd be easier for the user if each of the required parameters were placed at the top of the file. I understand that you've arranged the parameters in alphabetical order (outside of ```volumeType```, which makes sense), but this file feels slightly disorganized as is.

Reply:
I can move up the required parameters. I structured the variables to be in the same order as our existing [documentation](https://docs.aws.amazon.com/fsx/latest/APIReference/API_CreateFileSystem.html) (which I believe is in alphabetical order lol). It was mostly so I could check between the two and ensure I wasn't missing anything.